lib/android/audio.nit

```diff
@@ -377,6 +377,11 @@ class MediaPlayer
 
 	# Load a sound for a given resource id
 	fun load_sound(id: Int, context: NativeActivity): Music do
+		if playing then 
+			stop
+			reset
+			destroy
```

Comment:
need to be sure that the OS release the ressources used for the precedent sound ^^